.github/workflows/main.yml

```diff
@@ -9,11 +9,16 @@ on:
 jobs:
   build:
     strategy:
+      max-parallel: 1
```

Comment:
Is there a reason to not allow parallel jobs?

Reply:
There are 3 jobs in the CI, java11-test, java8-test, java17-test. Before adding this one, they are running parallel and the sequence of running is non-deterministic. One failure cancels the other. I want java11 to run first, so I specify the sequence of running [11, 8, 17] and requires max-parallel = 1.